src/style.css

```diff
@@ -0,0 +1,225 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700;800&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Open Sans", sans-serif;
+}
+
+.todo-list {
+  box-sizing: border-box;
+  border-radius: 3px;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.15);
+  margin: 80px auto;
+  top: 0;
+  max-width: 500px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+}
+
+.todo-list .todo-list-header {
+  display: flex;
+  width: 100%;
+  height: 50px;
+  background: #fff;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.08);
+}
+
+.todo-list .todo-list-header h1 {
+  color: #545862;
+  font-size: 15px;
+  font-weight: 400;
+  padding: 15px;
+  line-height: 20px;
+  margin: 0;
+  width: 92%;
+}
+
+.todo-list .todo-list-header .sync {
+  background-color: #fff;
+  color: #949090;
+  border: none;
+  width: 8%;
+}
+
+.todo-list .add-new-item {
+  width: 100%;
+  padding: 0;
+  background: white;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.08);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.todo-list .add-new-item .add-new-input {
+  width: 100%;
+  padding: 0 40px 0 1rem;
+  line-height: 50px;
+  height: 50px;
+  border: none;
+  font-style: italic;
+  font-weight: 300;
+  font-size: 15px;
+}
+
+.todo-list .add-new-item .add-new-input::placeholder {
+  font-family: "Open Sans", sans-serif !important;
```

Comment:
- [ ] It would be better to avoid using !important because it's bad practice and it makes debugging more difficult 👍 .